modules/consignment-api/database.tf

```diff
@@ -0,0 +1,59 @@
+resource "random_password" "password" {
+  length  = 16
```

Comment:
Probably worth increasing this, given that nobody's (hopefully) ever going to have to type it.

Reply:
I've put it up to 64. I can't find anything about password length limits for rds but this seems sensible.

Reply:
Apparently the max length is 41, I've set it to that.